packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/ChartConfigTabs.tsx

```diff
@@ -58,35 +56,40 @@ const ChartConfigTabs: FC = () => {
     const items = [...dimensionsInMetricQuery, ...metricsAndTableCalculations];
 
     return (
-        <Box w={320}>
-            <Tabs
-                onChange={setTab}
-                selectedTabId={tab}
-                renderActiveTabPanelOnly
-            >
-                <Tab
-                    id="layout"
-                    title="Layout"
-                    panel={<FieldLayoutOptions items={items} />}
-                />
-                <Tab
-                    id="series"
-                    title="Series"
-                    panel={<SeriesTab items={items} />}
-                />
-                <Tab
-                    id="axes"
-                    title="Axes"
-                    panel={<AxesOptions items={items} />}
-                />
-                <Tab
-                    id="legend"
-                    title="Display"
-                    panel={<LegendPanel items={items} />}
-                />
-                <Tab id="grid" title="Margins" panel={<GridPanel />} />
-            </Tabs>
-        </Box>
+        <Tabs w={335} defaultValue="layout">
+            <Tabs.List mb="sm">
+                <Tabs.Tab px="sm" value="layout">
```

Comment:
why do we need these ?

Reply:
The padding? It moves the tabs just a little closer together so that the dialog doesn't have to be even wider to accommodate them. It's a bit of a balance :)

Reply:
oh I see, so default is MD I guess?

Reply:
I think so. I think that's typically the case.